rockcraft.yaml

```diff
@@ -103,7 +107,7 @@ entrypoint-service: snapcraft
 services:
   snapcraft:
     override: replace
-    command: /bin/snapcraft [ pack ]
+    command: /bin/update-and-run.sh [ pack ]
```

Comment:
Maybe this should have a more generic name (e.g. `run-snapcraft`)? Just thinking any other setup and teardown work will probably live in that script too.

Reply:
good idea! updated